src/ResizeSensor.js

```diff
@@ -260,14 +260,16 @@
             }
         }
         else {
-            forEachElement(element, function(elem){
-                attachResizeEvent(elem, callback);
-            });
+            if (element !== undefined) {
```

Comment:
We have this check already in line 125.

Reply:
Oh nevermind :D

Reply:
Well, we should move that check to the very first line when we get "element", in the constructor.